src/utils/Processing.ts

```diff
@@ -168,6 +170,70 @@ export function writeFHIRResources(outDir: string, outPackage: Package, snapshot
   logger.info(`Exported ${count} FHIR resources as JSON.`);
 }
 
+/**
+ * Initializes an empty sample FSH within a user specified subdirectory of the current working directory
+ */
+export function init(): void {
+  logger.info(
+    'This tool will initialize a SUSHI project by creating all necessary files to build a simple IG. ' +
+      'The content of this IG will be simple default values, and should be updated. ' +
+      'The tool will now interactively create a config.yaml file for your project.'
+  );
```

Comment:
When running this, the "info" logger feels a little weird (since it's really a dialog).  I'd suggest we just straight-up `console.log` this.  Also, you will probably think I'm crazy but... I like it when the text pops out.  And you know how I like to pop text out, right?  Boxes!
```
╭──────────────────────────────────────────────────────────╮
│ This interactive tool will use your answers to create a  │
│ working SUSHI project configured with your project's     │
│ basic information. See the SUSHI documentation for more  │
│ guidance on project structure and configuration.         │
╰──────────────────────────────────────────────────────────╯
```